packages/kit-bg/src/dbs/local/LocalDbBase.ts

```
@@ -2749,18 +2728,101 @@
+    this.dbAllRecordsCache.set(cacheKey, devices);
+    return { devices };
+  }
+
+  async getAllWallets(): Promise<{
+    wallets: IDBWallet[];
+  }> {
+    const cacheKey = 'allDbWallets';
+    const allWalletsInCache = this.dbAllRecordsCache.get(
+      cacheKey,
+    ) as IDBWallet[];
+    if (allWalletsInCache && allWalletsInCache.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)